superchain/configs/mainnet/redstone.yaml

```diff
@@ -0,0 +1,22 @@
+name: redstone
+chain_id: 690
+public_rpc: https://rpc.redstonechain.com
+sequencer_rpc: https://rpc.redstonechain.com
+explorer: https://explorer.redstone.xyz
+
+superchain_level: 1
+
+batch_inbox_addr: 0xff00000000000000000000000000000000000690
+
```

Comment:
needs `plasma_config` in this file
see https://github.com/ethereum-optimism/superchain-registry/pull/224

Reply:
Added